engine/src/main/java/org/terasology/rendering/shader/ShaderParametersCombine.java

```diff
@@ -107,6 +107,12 @@ public void applyParameters(Material program) {
             program.setFloat("outlineThickness", outlineThickness, true);
         }
 
+        //TODO: Other parameters and volumetric fog test case is needed
+        Camera activeCamera = CoreRegistry.get(WorldRenderer.class).getActiveCamera();
+        if (activeCamera != null && CoreRegistry.get(Config.class).getRendering().isVolumetricFog()) {
+            program.setMatrix4("invViewProjMatrix", activeCamera.getInverseViewProjectionMatrix(), true);
```

Comment:
This is something I need to learn about: what does this line do? I mean, sure, I can see it sets a 4x4 matrix in the GPU to be the activeCamera's inverse view projection matrix. But why?